src/common/common.gyp

```diff
@@ -0,0 +1,26 @@
+{
+  'includes':[
+    'common.gypi',
+  ],
+  'targets': [{
+    'target_name': 'tizen_common',
+    'type': 'static_library',
+    'sources': [
+      'extension.cc',
+      'extension.h',
+      'logger.cc',
+      'logger.h',
+      'picojson.h',
+      'scope_exit.h',
+      'utils.h',
+      'XW_Extension.h',
+      'XW_Extension_EntryPoints.h',
+      'XW_Extension_Permissions.h',
+      'XW_Extension_Runtime.h',
+      'XW_Extension_SyncMessage.h',
+    ],
```

Comment:
I meant this new created static target `common.gyp:tizen_common` can be `set std=c++11` explicitly. Any extension shared_library target depend on this target, and use c++11 or c++0x in its own cflags.